docs/guides/integration-options-private-operator.md

```diff
@@ -51,33 +60,49 @@ The basic workflow is as follows:
 
 1. If an Operator is restarted, it goes through the attestation process again, and retrieves a fresh set of security information.
 
-1. The Operator re-attests periodically with the Core service to ensure that it is still running in a protected environment. If any compromise is detected, the Operator shuts down.
+1. The Operator re-attests periodically with the Core service to ensure that it is still running in a protected environment. If attestation fails, the Operator shuts down.
 
 ## Private Operator Security
 
 Each supported Private Operator implementation must meet rigorous security standards. Some security points include:
 
-- The Private Operator runs in a hardware-based trusted execution environment (TEE) hosted by one of the supported cloud providers listed in [Private Operator Integration Options](#private-operator-integration-options).
+- The Private Operator runs in a hardware-based trusted execution environment (TEE) hosted by one of the supported cloud providers listed in [Hosting Options for Private Operators](#hosting-options-for-private-operators).
 - The Private Operator must complete an attestation process before accessing the information needed to process UID2s.
 - The information on S3 is encrypted at rest and also encrypted in transit through TLS. In addition, access is limited to only correctly authorized and attested Private Operators.
 - The information retrieved at startup is not stored locally at any point. It is only ever held in memory, and the Private Operator is running in a protected environment that makes it difficult for anyone running the Operator (such as an Administrator), as well as any external players, to see the data that's in memory.
+- The Private Operator never stores <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> that is sent for processing (emails addresses and/or phone numbers). The data is only used within the enclave, to generate UID2s, and is discarded immediately after processing.
+
+## Private Operator Limitations
+
+There are a couple of limitations to Private Operator functionality:
+- Private Operators do not currently support generating the UID2 token on the client side.
```

Comment:
This should be: 
Private Operators do not currently support Client-side integration (with a link to the glossary item for Client-side integration).

Reply:
Done, thx.